vault/data_source_pki_secret_backend_config_cmpv2.go

```diff
@@ -63,6 +63,15 @@ func pkiSecretBackendConfigCMPV2DataSource() *schema.Resource {
 					Type: schema.TypeString,
 				},
 			},
+			consts.FieldDisabledValidations: {
```

Comment:
Every other field seems to have a "computed" line - why not this one?  How is this different than AuditFields in that sense?  I'm also not really sure how optional works here?  Most of these don't have that listed.

Reply:
The computed argument is mainly used when a value is returned from the server when no initial value from Terraform itself was provided. In this case since I believe the value returned will be an empty list it isn't required.

Optional only means the field does not need to be specified within the resource block